virt/virt-4-10-release-notes.adoc

```diff
@@ -95,6 +95,9 @@ The SVVP Certification applies to:
 //[id="virt-4-10-changes"]
 //== Notable technical changes
 
+//CNV-16642 - Dashboard feature.
+* The {VirtProductName} dashboard provides data on resource consumption for virtual machines and associated pods. The visualization metrics displayed in the {VirtProductName} dashboard are based on xref:../virt/logging_events_monitoring/virt-prometheus-queries.adoc#virt-prometheus-queries[Prometheus Query Language (PromQL) queries].
```

Comment:
A couple of phrasing suggestions for you to consider:

s/provides data on resource consumption/displays resource consumption data

s/The visualization metrics displayed/The metrics that are represented

Reply:
Plan to incorporate the first suggestion. I am not seeing much  value gained by changing the original phrasing for the second item, therefore retaining it.